examples/factory-contract/README.md

```diff
@@ -1,3 +1,75 @@
-# Factory contract
+# Factory Contract
 
-TBD
+Example implementation of a factory contract using [near-sdk-rs].
+
+[near-sdk-rs]: https://github.com/near/near-sdk-rs
+
+NOTES:
+
+- This example demonstrates how to create and manage multiple instances of a contract from a factory contract.
+- Ensure that the factory contract is deployed and accessible.
+
+## Project Structure
+
+- `.cargo`: Configuration files for Cargo, the Rust package manager.
+- `high-level/src`: Source code for the high-level factory contract example.
+- `low-level/src`: Source code for the low-level factory contract example.
+- `res`: Compiled contract binaries.
+- `tests`: Integration tests for the factory contract examples.
+
+## Building
+
+To build run:
+
+```bash
+./build.sh
+```
+
+## Testing
+
+To test run:
+
+```bash
+cargo test --package factory-contract -- --nocapture
+```
+
+## Usage
+
+## Usage
+
+1. Prerequisite: Install cargo-near:
+
+```bash
+cargo install cargo-near
+```
+
+2. Create a new testnet account:
+
+```bash
+cargo near create-dev-account
+```
+
+3. Deploy the factory contract:
+
+```bash
+cargo near deploy --wasmFile path/to/factory-contract.wasm --accountId your-account.testnet
+```
+
+4. Use the factory contract to create instances of the target contract:
+
+```bash
+near call your-factory-contract-account.testnet create_instance '{"args": "value"}' --accountId your-account.testnet
```

Comment:
This command is outdated, please, use the new syntax (it is suggested to you in the output):

```suggestion
near contract call-function as-transaction your-factory-contract-account.testnet deploy_status_message json-args '{"account_id": "sub.your-factory-contract-account.testnet"}' prepaid-gas '100 Tgas' attached-deposit '0 NEAR'
```